src/riscv.h

```diff
@@ -228,12 +383,20 @@ typedef struct {
     char *elf_program;
 } vm_user_t;
 
+typedef struct {
+    char *elf_program;
```

Comment:
It is weird that bot `vm_user_t` and `vm_system_t` have only a member `elf_program`. If so, why another structure?

Reply:
Since I am loading a single ELF program for MMU testing and I should put a FIXME there to clarify. Eventually, we have to put kernel image, device tree blob, etc in the structure.